lib/bike_brigade_web/live/components/rider_selection_component.ex

```diff
@@ -62,13 +62,26 @@ defmodule BikeBrigadeWeb.Components.RiderSelectionComponent do
      |> assign(:search, "")}
   end
 
+
+
+  def handle_event("load_more", _values,%{ assigns: assigns } = socket) do
+    %{ rider_search: rider_search, riders: riders } = assigns
+    next_page_rs= RiderSearch.next_page(rider_search)
+    {rider_search, search_results} = RiderSearch.fetch(next_page_rs)
+     {:noreply,
+     socket
+     |> assign( :riders, riders ++ search_results.page )
+     |> assign(:rider_search, rider_search)
+     }
+  end
+
   defp fetch_results(socket) do
     {rider_search, search_results} =
       RiderSearch.fetch(socket.assigns.rider_search, socket.assigns.search_results)
 
     socket
     |> assign(:rider_search, rider_search)
-    |> assign(:search_results, search_results)
+    |> assign(:riders, search_results.page)
```

Comment:
I think we want to also update the assigns on mount we have `     |> assign(:search_results, %RiderSearch.Results{})` but it should be `     |> assign(:riders, [])`

It would cause an error if we tried to access riders before its set, so good to assign the defaults.